### PR TITLE
Use imperative mood in changelog entry

### PR DESCRIPTION
**Pull request by andrross**
Apologies in advance for being a pedant, but we in the OpenSearch project like to keep our changelog entries consistent and the dependabot changelog helper is an outlier here. Matching the style of the commit message from dependabot seems appropriate, but please feel free to let me know if you disagree :)

---

The imperative mood is generally recommended for git commit messages. Git uses it for things like "Revert ..." or "Merge ..." and dependabot uses it in the [commit message][1]. This changes the changelog helper to use the imperative mood for consistency.

[1]: https://github.com/dangoslen/dependabot-changelog-helper/commit/11616d35c16f6141349e2417035d9440c3218b36

### CODE REVIEW

**Comment by andrross** [comment]
> ### Dependencies
> * Bumps `typescript` from 4.9.3 to 4.9.4
> * Bumps `actions/checkout` from 3.1.0 to 3.3.0
> * Bumps `prettier` from 2.8.0 to 2.8.3
> * Bumps `@types/node` from 14.18.33 to 18.11.18
> * Bumps `eslint-plugin-jest` from 27.1.6 to 27.2.1
> * Bumps `eslint` from 8.28.0 to 8.32.0

Obviously this PR would result in a bit of a jarring change in existing CHANGELOGs...

File: src/changelog-updater.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ function buildEntryLine(entry: DependabotEntry): string {
 }
 
 function buildEntryLineStart(entry: DependabotEntry): string {
-  return `- Bumps \`${entry.package}\` from`
+  return `- Bump \`${entry.package}\` from`
```
Comment (dangoslen) [inline src/changelog-updater.ts:74]: I think this makes a lot of sense.

I wonder if this would be worthing making a configuration option for other users?

**Comment by dangoslen** [comment]
@andrross not being a pedant at all. I think it could be a good idea to make this a configuration option of `entryLinePrefix` that defaults to `- Bump` 

I do think we would need to mark this as a breaking change since it changes the behavior and I could see other teams wanting the opposite.

**Comment by andrross** [comment]
> @andrross not being a pedant at all. I think it could be a good idea to make this a configuration option of `entryLinePrefix` that defaults to `- Bump`
> 
> I do think we would need to mark this as a breaking change since it changes the behavior and I could see other teams wanting the opposite.

I agree with all of this. It's definitely a breaking change but I like the idea of making it the new default. Configuration options do add complexity but it seems likely there will be other users who prefer the original so giving them the option will be important. There may even be other pedants who prefer `- Update` or something else :)

**Comment by dangoslen** [comment]
Yep - I had thought about this early on (even wrote an [issue for it](https://github.com/dangoslen/dependabot-changelog-helper/issues/28)), but decided I'd rather get something out.

And while I'd like to create a more general "format string" pattern, there are some complications around finding matching lines. Using the idea of a prefix might be the easiest way to give _some_ control, but not have to rethink several aspects of the tool (which I'm already working on a refactor or too).

If you want to add the config option here I'm happy to collab on it! Or also happy to accept these changes as is and I can work adding that config before creating a `3.0` release

File: __tests__/changelog-updater.test.ts [hunk 2]
```diff
@@ -364,19 +364,80 @@ test('Updates existing section when between other sections', async () => {
...
+
+  await updateChangelog(PACKAGE_ENTRY, 'v1.0.0', './CHANGELOG.md', 'Update')
+
+  const params = fs.writeFileSync.mock.calls[0]
+  expect(params[0]).toStrictEqual('./CHANGELOG.md')
+  expect(params[1]).toStrictEqual(`# Changelog
+
+## [v1.0.0]
+### Dependencies
+- Bump \`package\` from v1 to v2`)
```
Comment (andrross) [inline __tests__/changelog-updater.test.ts:418]: Note there is some subtlety here regarding finding existing entries/duplicates. I've chosen _not_ to change the prefix word of an existing entry if a different prefix word is provided.

**Comment by dangoslen** [comment]
@andrross I think this looks good! Thanks for the contribution.

 I do have a patch release in another PR that I will merge later tonight/tomorrow morning and then will get to merging this later this week. 

You PR is forcing me to do the clean-up I've been needing to do, so apologies that this won't get super fast, but I'd like to get some dependencies updated and the CI cleaned back up before doing a v3.0. Is that acceptable to you?

**Comment by andrross** [comment]
> You PR is forcing me to do the clean-up I've been needing to do, so apologies that this won't get super fast, but I'd like to get some dependencies updated and the CI cleaned back up before doing a v3.0. Is that acceptable to you?

Yeah, no problem at all!

**Comment by dangoslen** [comment]
Hey @andrross after merging the patch PR and upgrading some deps, there looks to be a view conflicts. I don't think they are too bad though. Do you mind fixing those up?

For the `dist/index.js.map` file, you can just accept the incoming changes and I'll generate that file when I cut the release.

**Review by dangoslen — APPROVED** [review]

**Pull request merged**